bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -2,60 +2,72 @@
 # in ../docs/source/user/versions.rst
 # so please keep that documentation up to date as they change
 
-# basics
-python>=3.7
-conda=23.3.*
-boa=0.15.*
-conda-build=3.24.*
-conda-verify=3.1.*
-argh=0.26.*          # CLI
-colorlog=4.8.*       # Logging
-tqdm>=4.26           # Progress monitor
-yaspin =2
-ruamel_yaml=0.15.*   # Recipe YAML parsing
-pyaml=17.12.*        # Faster YAML parser (deprecate?)
-networkx=2.*
-pandas=1.4.*
-numpy=1.19.*         # Avoid breaking pandas on OSX
-libblas=*=*openblas  # Avoid large mkl package (pulled in by pandas)
-boltons=23.*
-jsonschema=3.2.*     # JSON schema verification
-pyopenssl>=22.1      # Stay compatible with cryptography
-
 # pinnings
 conda-forge-pinning=2023.05.06.13.08.41
 
-# tools
-anaconda-client=1.12.*  # anaconda_upload
-involucro=1.1.*        # mulled test and container build
-skopeo=1.11.*          # docker upload
-git=2.*                # well - git
+# basics
+python=3.10.*
+conda=24.3.*
+conda-libmamba-solver=24.1.*
+conda-build=24.3.*
+conda-index=0.4.*
+mamba=1.5.*
+boa=0.17.*
+
+argh=0.31.*                   # CLI
+colorlog=6.*                  # Logging
+tqdm=4.66.*                   # Progress monitor
+yaspin=2.*                    #
+ruamel.yaml=0.18.*            # Recipe YAML parsing
+pyaml=23.12.*                 # Faster YAML parser (deprecate?)
+networkx=3.2.*                #
+pandas=2.2.*                  #
+libblas=*=*openblas           # Avoid large mkl package (pulled in by pandas)
+boltons=23.*                  #
+jsonschema=4.21.*             # JSON schema verification
+jinja2=3.1.*                  #
+
+anaconda-client=1.12.*        # anaconda_upload
+galaxy-tool-util=23.*         # mulled test and container build
```

Comment:
Would it be OK to update `galaxy-tool-util` to 24.0.0 ?

Reply:
There are a few deps that got updated in the meantime; I'm looking at their changes right now.